backend/views.py

```diff
@@ -0,0 +1,8 @@
+from canvas_oauth.oauth import get_oauth_token
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render
+
+@login_required
+def get_home_template(request):
+    access_token = get_oauth_token(request)
```

Comment:
Seems to be, when I glanced at their repo.

Reply:
Yeah, this is the pattern they want to use. Protect every route with login_required and get_oauth_token. Calling it at this point (get_home_template) will trigger the authorization right at the start of use of the app and the token available for use later. It will be needed here to get the list of placements as part of the next PR.